pynicotine/gtkgui/search.py

```diff
@@ -566,6 +566,7 @@ def __init__(self, searches, text, token, mode, mode_label, room, users, show_pa
             ("#" + _("Download _Folder(s)"), self.on_download_folders),
             ("#" + _("Download F_older(s) To…"), self.on_download_folders_to),
             ("", None),
+            ("#" + _("_View User Profile(s)"), self.on_user_profile),
```

Comment:
`"_View User Profile(s)"` -> `"_View User Profile"`

Suggest hide or disable the menu item if multiple users are selected, because it is not useful to open multiple User Profile tabs.

The menu item visibility/sensitivity could be set on opening of the context menu, at the same time as the User Actions menu(s) are built..

Reply:
> Suggest hide or disable the menu item if multiple users are selected

Not necessary in my opinion, we can just open the user profile of the first selected user.

Reply:
Fair enough, that is a lot more straight-forward than my suggestion.